extending-valtimo/access-control/create-relation-between-resources.md

```diff
@@ -0,0 +1,96 @@
+# Creating a relation to another resource
```

Comment:
This page is not reachable right now. I think this should at the very least be included in `SUMMARY.md`